handler/resource.go

```diff
@@ -42,11 +42,11 @@ func createResource(cfg *config.Resource) (resource.Resource, error) {
 	case "httpclient":
 		return httpclient_r.New(cfg)
 	case "httpserver":
```

Comment:
this should be `wiremock`, and you map `wiremock` as `http/server` on `resources` on line 22. that way we can use `wiremock` as resource type on tomato config